dali/dfu/dfuutil.cpp

```diff
@@ -1296,6 +1296,24 @@ class CDFUhelper: implements IDFUhelper, public CInterface
         }
     }
 
+    MemoryBuffer &getFileHistory(const char *lfn,MemoryBuffer &out, IUserDescriptor *user)
```

Comment:
But in fact this method should be here.
DFUhelper/dfuplus is for commands that interact with esp to perform DFU/DFS work.

This is only looking up a file and returning the history AND where it's used is already in Esp.
It could be private utility method where used though.